base/tuple.jl

```diff
@@ -565,6 +565,15 @@ function isless(t1::Any32, t2::Any32)
     return n1 < n2
 end
 
+# Performance optimization to pack pairs of integers into a single integer
+# for comparison. This is useful for sorting tuples of integers.
+# TODO: remove this when the compiler can optimize the generic version better
+# See #48724 and #48753
```

Comment:
```suggestion
```

Reply:
I added this TODO to it make it clear to future readers that it is okay to remove this if it no longer produces measurable performance improvements. @vtjnash, why do you prefer to remove it?